src/agent/agent.c

```diff
@@ -59,6 +59,39 @@ typedef struct {
         char *method;
 } AgentJobOp;
 
+int agent_cfg_set_defaults(struct config *config) {
```

Comment:
I'd prefer to have default configuration stored in `libhirte/common/cfg.c`. There can be 2 methods:

- `cfg_agent_def_conf(struct config *config)` - populating the default configuration for agent
- `cfg_manager_def_conf(struct config *config)` - populating the default configuration for manager

Then in those methods we will fill in the default values for all valid option for each component. And this method could also populate the default section.